backend/api/Controllers/Models/CustomMissionQuery.cs

```diff
@@ -8,7 +8,7 @@ public struct CustomInspectionQuery
 
         public float? VideoDuration { get; set; }
 
-        public string? AnalysisTypes { get; set; }
+        public AnalysisType AnalysisTypes { get; set; }
```

Comment:
```suggestion
        public AnalysisType? AnalysisType { get; set; }
```
CustomInspectionQuery should reflect Inspection